src/Sylius/Bundle/MoneyBundle/DependencyInjection/SyliusMoneyExtension.php

```diff
@@ -32,11 +30,27 @@ public function load(array $config, ContainerBuilder $container)
         $config = $this->processConfiguration($this->getConfiguration($config, $container), $config);
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
 
-        $container->setParameter('sylius.money.locale', $config['locale']);
-        $container->setParameter('sylius.money.currency', $config['currency']);
+        $container->setParameter('sylius_money.locale', $config['locale']);
+        $container->setParameter('sylius_money.currency', $config['currency']);
```

Comment:
Do we need to add `sylius_money.currency` parameter? Are there any drawbacks of simpyl using `currency` parameter?